layouts/updates/list.html

```diff
@@ -14,8 +14,20 @@
     <tr>
       <td>{{ .Title }}</td>
       <td>{{ .Date.Format "2006-01-02" }}</td>
-      <td><a href='{{ .Params.update }}'>Download <code>update.tar.xz</code></a></td>
-      <td><a href='{{ .Params.image }}'>Download <code>pi-image.img.xz</code></a></td>
+      <td>
+        {{ if .Params.update }}
+        <a href='{{ .Params.update }}'>Download <code>update.tar.xz</code></a>
```

Comment:
Could we either indent this line one more so that it matches the "No update file available" text, or dedent that line to match this one? (The same comment also applies to the image file below)

I'm not sure what our usual guidance is for intending inside template blocks; do we have a preference?

Reply:
My preference is to consider template tags like this to create a new block, and so should be indented. Will update